go.mod

```diff
@@ -35,6 +34,7 @@ require (
 	github.com/onsi/gomega v1.10.0
 	github.com/oschwald/maxminddb-golang v1.5.0 // indirect
 	github.com/pborman/uuid v1.2.0 // indirect
+	github.com/pkg/errors v0.8.1
```

Comment:
Is this required bc of Geth? For some reason I feel like at some point we accidentally required this at some point when we really wanted the core `errors` package

Reply:
🤔  Not sure why that ended up in there, don't think it's necessary though.